.github/workflows/ci.yaml

```
@@ -15,9 +15,9 @@ jobs:
           - 'ubuntu-latest'
 #         - 'macos-latest'
         ocaml-version:
-          - 4.09.1
-          - 4.10.1
-          - 4.11.1
+          - 4.10.2
```

Thanks!  But I'd like to test against 4.09 as long as the opam file declare “this project can be compiled with OCaml >= 4.09.0”

```suggestion
          - 4.09.1
          - 4.10.2
```